include/document.hpp

```diff
@@ -29,7 +29,7 @@ namespace kuzzleio {
             Document(Kuzzle* kuzzle);
             Document(Kuzzle* kuzzle, document *document);
             virtual ~Document();
-            int count(const std::string& index, const std::string& collection, const std::string& body, query_options *options=nullptr);
+            int count(const std::string& index, const std::string& collection, const std::string& body="", query_options *options=nullptr);
```

Comment:
I think a more elegant way to do this is to expose a second method  that don't have the `body` parameter:
```suggestion
            int count(const std::string& index, const std::string& collection, const std::string& body, query_options *options=nullptr);
            int count(const std::string& index, const std::string& collection, query_options *options=nullptr);
```
The second one could call the first one with body set to "", and this would avoid the user having to call the method
with an empty string "" when he wants to pass some `query_options`.